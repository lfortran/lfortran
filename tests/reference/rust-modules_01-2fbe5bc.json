{
    "basename": "rust-modules_01-2fbe5bc",
    "cmd": "lfortran --no-color --show-rust {infile}",
    "infile": "tests/../integration_tests/modules_01.f90",
    "infile_hash": "17c761eb72726e0dde59f2b21f98f2402d783613834e927c49047534",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "rust-modules_01-2fbe5bc.stdout",
    "stdout_hash": "fbcfa1bd43feaa4d552942310c801ba3dfca464be11f00a7ae1303cf",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}