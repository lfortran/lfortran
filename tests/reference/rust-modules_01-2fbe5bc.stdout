use num::complex::Complex;
use num::traits::Pow;
use ndarray::{array, Array1, Array2, Array3, Array4, Array5, Array6};

mod a_01 {
pub fn b() {
    println!("{}", "b()".to_string());
}

}

use a_01::{b};

fn main() {
    b();
}
